### ISSUE TITLE
Error when installing "pip install scikit-network"

### ISSUE THREAD

**Issue by wanghao98** [sknetwork-team/scikit-network#573]
* scikit-network version: NA
* Python version: Python 3.12.3
* Operating System: Ubuntu 22.04.1 LTS

### Description

I was trying to install a package, which the scikit-network is an requirement. However, I hit an error, listed below

### What I Did

```
pip install scikit-network

      error: command '/usr/bin/gcc' failed with exit code 1
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for scikit-network
  Running setup.py clean for scikit-network
Failed to build scikit-network
ERROR: Could not build wheels for scikit-network, which is required to install pyproject.toml-based projects

```
please help. Thanks!

**Comment by tbonald** [comment]
Hello,
scikit-network is not yet available in Python 3.12.
Coming soon...
Best

**Comment by tbonald** [comment]
Python 3.12 is available on the Develop branch. 
A new release will be published soon.
